Interview API Project/Service/UserService.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Interview_API_Project.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Interview_API_Project.Service
+{
+
+    public class UserService
+    {
+        private List<User> _users = new List<User>
+        {
+            new User { Id = 1, Name = "John Doe", Email = "john@example.com", IsActive = true },
```

Comment:
in a production environment, we would want to set the _users list based on the data coming in instead of hard-coded